src/main/flight/imu.c

```diff
@@ -80,10 +83,9 @@ STATIC_FASTRAM float smallAngleCosZ;
 
 STATIC_FASTRAM bool isAccelUpdatedAtLeastOnce;
 
-STATIC_FASTRAM_UNIT_TESTED float q0, q1, q2, q3;    // quaternion of sensor frame relative to earth frame
-STATIC_FASTRAM_UNIT_TESTED float rMat[3][3];
-
+FASTRAM fpQuaternion_t orientation;
 FASTRAM attitudeEulerAngles_t attitude;             // absolute angle inclination in multiple of 0.1 degree    180 deg = 1800
+STATIC_FASTRAM_UNIT_TESTED float rMat[3][3];
```

Comment:
I intend to eventually phase out `rMat` so I don't want to touch it at all.

Reply:
yes ... 
converting quaternion to matrix + matrix multiplication is fastest way to do quaternion rotation, even when transforming single vector.
In imu, conversion is amortized, so using 'temporary' matrix is nice and quite cheap optimization.

Another use of mat34 is sensor calibration - it can represent all calibration transformations (sensor alignment, axis scaling, offset, cross-axis sensitivity) in compact and mathematically sound way ...

Reply:
Ok, worth doing than. Will look into it in a separate PR

Reply:
Just keep this option open,in can be implemented later ...